schemas/app-sre/slo-document-1.yml

```diff
@@ -26,7 +26,15 @@ properties:
         namespace:
           "$ref": "/common-1.json#/definitions/crossref"
           "$schemaRef": "/openshift/namespace-1.yml"
-        targetNamespace:
+        SLONamespace:
+          description: |
+            This field is used to determine in which namespace to generate the PrometheusRules
+            that contain the dynamic alerts generated for the SLO, that are being used in the
+            slo-to-status-board flow:
```

Comment:
What about something like?

```
This field is used to determine in which namespace PrometheusRule object
that contain the alerts generated from the expressions of this SLO document
is going to be created. Those alerts are being used in the slo-to-status-board 
flow:
```

Reply:
"is going to be created" can be is located. The key point is to avoid the feeling that something will automagically create the PrometheusRule

Reply:
👍 applied